src/database/queries/applicants/selectByExternalId.sql

```diff
@@ -0,0 +1,6 @@
+SELECT a.id AS "id",
+  a.external_id as "externalId",
+  a.opted_in AS "optedIn",
+  a.created_at AS "createdAt"
+FROM applicants a
+WHERE a.external_id = :externalId;
```

Comment:
It's a matter of preference, I suppose. I assume most queries have a join somewhere and then for consistency we use the aliases everywhere. Perhaps this style change could go into a separate PR that changes all of our queries to have a consistent look/feel?

Reply:
I opened #321 to start that discussion!

Reply:
I pushed a change to remove aliasing where it was unneeded, also SHOUTED "AS" MORE CONSISTENTLY! I am OK with shouting SQL but I think it's a funny convention we happened upon.

Reply:
I never stopped to think about the shouting but you're right 🤣

Reply:
tbh I love the shouting. It's one of those things that makes SQL feel like SQL to me. Maybe because it's "SQL" and not "sql"? 🤔 ← @slifty can we get a :thonking: here plz

Reply:
When y'all are using `psql` do you really hold shift or frequently tap caps lock when querying the database?

Reply:
@bickelj I do indeed, truly! (I don't say that as a form of advocacy, just describing my personal pathology.)